gcloud-java-storage/src/main/java/com/google/gcloud/storage/BatchResponse.java

```diff
@@ -133,7 +135,8 @@ public boolean equals(Object obj) {
     BatchResponse other = (BatchResponse) obj;
     return Objects.equals(deleteResult, other.deleteResult)
         && Objects.equals(updateResult, other.updateResult)
-        && Objects.equals(updateResult, other.updateResult);
+        && Objects.equals(updateResult, other.updateResult)
```

Comment:
Lets fix the already existing bug and replace duplicate `updateResult` with`getResult` :)